data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/KafkaHealthIndicator.java

```diff
@@ -35,7 +35,7 @@
 import java.util.stream.Collectors;
 
 /**
- * Kafka health indicator. Check that the consumer group can be accessed and is registered with the cluster,
+ * Kafka health indicator checks that the consumer group can be accessed and is registered with the cluster,
```

Comment:
We're not checking the consumer group as this service does not have a consumer. You could check how the palisade or audit service document this.

Reply:
Ok,   rolled it back to what it was.